Sming/Components/crypto/include/Crypto/Md5.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "HashContext.h"
+
+namespace Crypto
+{
+using Md5Hash = HashValue<MD5_SIZE>;
```

Comment:
Is this necessary? It creates an alternative to `Md5::Hash`, which raises the question which one is preferred.

Reply:
I'd prefer not to have this but ran into difficulty having it defined directly inside the class. Suggestions?

Reply:
I'll pass the hash size to the class template instead, thanks.

Reply:
Could you please elaborate on the problems you encountered?
I would have expected `using Hash = HashValue<MD5_SIZE>;` inside the class should work fine.

Reply:
I've re-jigged using your Engine idea, much better. This issue has now evaporated :-)